bin/setup

```diff
@@ -60,18 +60,6 @@ create_link() {
   fi
 }
 
-#######################################
-# Runs the fzf install script in order to install key bindings and shell completion. Exits with an
-# error code of 1 if that script fails.
-#######################################
-setup_fzf() {
```

Comment:
`fzf` may not be installed at this point. We've moved the installation to a point when we know `fzf` is installed.